### DISCUSSION TITLE
I can't update

### DISCUSSION THREAD

**Discussion by Homer-Max** [mylinuxforwork/dotfiles#229] [General]
![photo_2024-09-09_08-51-44](https://github.com/user-attachments/assets/529c539b-be4e-4a8d-ba9c-4824b1f9e88d)

I've tried for many times since yesterday

**Comment by AnthonyBAC** [comment]
same here

**Reply by Homer-Max** [reply]
I have fixed it.
rm ~/.cache/yay/ml4w-hyprland/*.part 
and try it again

**Reply by AnthonyBAC** [reply]
it says that doesnt existe that directory 

![image](https://github.com/user-attachments/assets/13320a19-7bcd-4cc3-a752-1bbd6cea82ce)

**Reply by Homer-Max** [reply]
copy your former error to ask claude3.5 or gpt4o

**Reply by mylinuxforwork** [reply]
Will look into it.

**Reply by mylinuxforwork** [reply]
@AnthonyBAC @Homer-Max Just installed it without any problems with yay and paru. I cleaned up my system before with the welcome app/settings/system Cleanup your system

If you update the ml4w dotfiles, can you try it as well?
